src/main/webapp/resources/js/pages/projects/processing/ProcessingAutomatedPipelines.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { AutomatedPipelineHeader } from "./AutomatedPipelineHeader";
+import { AnalysisTemplates } from "./AnalysisTemplates";
+import { ProcessingPriorities } from "./ProcessingPriorities";
+
+export function ProcessingAutomatedPipelines({ projectId, canManage }) {
+  return (
+    <section>
+      <AutomatedPipelineHeader canMange={canManage} projectId={projectId} />
+      {canManage && <ProcessingPriorities projectId={projectId} />}
```

Comment:
Indentation seems off on this line

Reply:
Prettier runs automatically so I think it is ok, I did check though

Reply:
Sounds good